### ISSUE TITLE
Decide on location for DAP AVT1 reports

### ISSUE THREAD

**Issue by joshblack** [carbon-design-system/carbon#2298]
We should regularly run AVT1 tests using DAP on our components and should store the results in a location that's accessible by anyone using Carbon. This issue is to write up the process for these AVT1 results and start the process of surfacing that information as appropriate.

cc @dakahn

**Comment by dakahn** [comment]
Agreed! These results should be accessible in our websites

**Comment by dakahn** [comment]
This has become [part of a larger specification effort](https://github.com/carbon-design-system/carbon-spec) that we're currently pluggin away at. Closing this issue.
